docs/spark/spark-writes.md

```diff
@@ -311,7 +311,9 @@ distribution & sort order to Spark.
 {{< /hint >}}
 
 {{< hint info >}}
-Both global sort (`orderBy`/`sort`) and local sort (`sortWithinPartitions`) work for the requirement.
```

Comment:
I think for this we should note above that this is not required for recent spark releases and the information below will come in conflict with a built in table Sort Order or distribution. (it should not override a built in ordering)

Reply:
Sorry I don't know much about this part yet, I may need to research it first.

Reply:
That's fine we probably can do a separate doc to actually explain the Distribution/Ordering built in's if we haven't yet

Reply:
Yes, that sounds great.

Reply:
Yes, that sounds great.